tests/UKHO.ERPFacade.API.FunctionalTests/FunctionalTests/SapCallbackEndpointScenarios.cs

```diff
@@ -0,0 +1,34 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using UKHO.ERPFacade.API.FunctionalTests.Service;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.FunctionalTests
+{
+    [TestFixture]
+    public class SapCallbackEndpointScenarios
+    {
+        private SapCallbackEndpoint _sapCallbackEndpoint;
+
+        [SetUp]
+        public void Setup()
+        {
+            _sapCallbackEndpoint = new SapCallbackEndpoint();
+        }
+
+        [Test]
+        public async Task WhenSAPEndpointHitWithNonExistingCorrelationId_ThenSAPShouldReturns404NotFoundResponse()
```

Comment:
Please correct the test case name. It should be CallbackEndpoint who returns the status codes not SAP.